opal/mca/threads/pthreads/threads_pthreads_wait_sync.h

```diff
@@ -15,6 +15,7 @@
  * Copyright (c) 2015-2016 Research Organization for Information Science
  *                         and Technology (RIST). All rights reserved.
  * Copyright (c) 2019      Sandia National Laboratories.  All rights reserved.
+ * Copyright (c) 2021      Argonne National Laboratory.  All rights reserved.
```

Comment:
Why adding the copyright header here? I don't see any changes.

Reply:
Thank you. It was my mistake. I fixed it.